### DISCUSSION TITLE
Uncaught DomainException in Printer.php:37

### DISCUSSION THREAD

**Discussion by rus0228** [php-aidc/label-printer#5] [Q&A]
Hi everyone,
For printing our product label, we are determining to use this package because this one would be powerful in our case. So created spike shortly and run it on the local connecting to the printer, but I am getting error as following: 
```
Fatal error: Uncaught DomainException in vendor/php-aidc/label-printer/src/Printer.php:37
```
I am not sure what's happened because it's first time to use this one for me.
That would be appreciated with your helps.
Thanks.

**Comment by jhaoda** [comment]
Please, provide code that cause this error.
